src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate.java

```diff
@@ -556,41 +573,53 @@ public String getDisplayName() {
         return placementStrategies;
     }
 
+    /**
```

Comment:
is that changing the current behavior here?
not sure how much it is is in use, but if so its sth that we should explicitly mark in release notes

Reply:
Sorry, just seeing this. I can comment on everything if it matters

Reply:
The only thing I added was to `setEntryPoint` and add static imports to reduce the verbosity. But, I did notice the inconsistency in usage and override of entryPoint. Is there any reason to not allow the override of entrypoint?

Reply:
i don't think its in very wide use, but if sth changes for the endusers we should put that at least in the release notes to make sure people are aware

Reply:
Well, I could support it easily enough - I only discovered it wasn't working because I tried to make use of it in a test. It's one of the properties you can override in the step, but it's not carried forward to the task definition. 

For sake of limiting this work, I will simply update the documentation to say it's not supported